2017-12-06-invitacion-meetup.md

```diff
@@ -0,0 +1,17 @@
+¡Hola Javascript@s!
+
+Se acerca nuestro próximo meetup "[Star Vars: The Last Promises](https://www.meetup.com/es-ES/NodersJS/events/239484529/)"  el **14 de diciembre**. En esta oportunidad son nuestros amigos de Groupon Latam quienes nos recibirán en sus oficinas.
```

Comment:
`The Last Promise` - `Groupon LatAm' (ojo con las mayúsculas/minúsculas del latam, es parte de su brand)